flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/DefaultExecutionGraphBuilder.java

```diff
@@ -121,6 +121,14 @@ public static DefaultExecutionGraph buildGraph(
         // create a new execution graph, if none exists so far
         final DefaultExecutionGraph executionGraph;
         try {
+            TaskDeploymentDescriptorFactory taskDeploymentDescriptorFactory =
```

Comment:
I think it would be better to add a separate try catch block, because the exception message of current block is "Could not create the ExecutionGraph"